controllers/PostController.js

```diff
@@ -42,7 +42,7 @@ const addReply = async (req, res) => {
 
 const getPosts = async (req, res) => {
   try {
-    const { userId } = req.query;
```

Comment:
Good catches, I went through and made the suggested changes and verified that everything else seems up to standard. Additional thing I noticed is that the `getFollowStatus` query and route is not actually being called on the frontend at all so it seems like we've deprecated that query. I decided to remove the route, controller and query because I couldn't find a reason to keep it. I think I initially made this query but I just used `getProfileContents` to pass that over instead of querying for it separately